test/spec/upload.js

```diff
@@ -14,6 +14,18 @@ if (isNode) {
   var Blob = Buffer;
 }
 
+var getStorage = function () {
+  // this storage is only needed for the node environment. It defaults to localStorage
+  // for browser.
+  if (isNode) {
+    var temp = require("temp");
+    var storagePath = temp.path();
+    return new tus.FileStorage(storagePath);
```

Comment:
Please add code to empty the storage after every test to ensure that each unit test is run in isolating. This is already done for localstorage in the browser: https://github.com/tus/tus-js-client/pull/73/files#diff-893840c5db7894faca0e2e7ea5a7020dR47

Reply:
there's no need to do this for `FileStorage` because we are always calling `var temp = require("temp");` which creates a separate storage file.

Reply:
Ok, fair enough.